Logs rate limit control documentation (#5666)
DataDog/documentation: content/en/logs/indexes/_index.md

{{< img src="logs/indexes/enable_index_filters.png" alt="enable index filters" responsive="true" style="width:80%;">}}

### Set daily quota

> You can set a daily quota to hard-limit the number of logs that are stored within an Index per day. This quota is applied for all logs that should have been stored (i.e after exclusion filters are applied). Once the daily quota is reached, logs are not indexed anymore and go to your Archives if those are set.

Reviewer: ```suggestion
You can set a daily quota to hard-limit the number of logs that are stored within an Index per day. This quota is applied for all logs that should have been stored (i.e after exclusion filters are applied). Once the daily quota is reached, logs are no longer indexed and go into your Archives if those are set.
```

Update or remove this quota at any time when editing the Index: 

{{< img src="logs/indexes/index_quota.png" alt="index details" responsive="true" style="width:70%;">}}

> **Note**: Indexes quota resets automatically at 12:00pm UTC (2:00pm CET, 8:00am EDT, 5:00am PDT).

Reviewer: ```suggestion
**Note**: Indexes quota resets automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
```
